### PR TITLE
Corrected problem with siblingId

### PR DESCRIPTION
**Pull request by sonofmun**
I am still getting this error every once in a while. Could you merge this, @PonteIneptique , and draft a release.

### CODE REVIEW

File: .directory [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Dolphin]
```
Comment (PonteIneptique) [inline .directory]: Can you remove the file from there >

File: python3.5 [hunk 2]
```diff
@@ -0,0 +1 @@
+/usr/bin/python3.5
```
Comment (PonteIneptique) [inline python3.5]: same here

File: notebooks/Untitled.ipynb [hunk 3]
```diff
@@ -0,0 +1,477 @@
+{
```
Comment (PonteIneptique) [inline notebooks/Untitled.ipynb]: and maybe here as well :)

**Comment by PonteIneptique** [comment]
LGTM except the supplementary files

File: notebooks/Untitled.ipynb (hunk 3 above)
Comment (sonofmun) [reply notebooks/Untitled.ipynb]: That's what happened to my Jupyter Notebook!

**Comment by PonteIneptique** [comment]
I'll be riding my bike in a minute, so when I am at the office, I'll squash and releae

**Comment by PonteIneptique** [comment]
Did you just push to dev directly ? O_O\
Damn, I can't squash then...

**Comment by sonofmun** [comment]
Yeah. I only realized that I was pushing directly to dev after I had already done it. Maybe you should protect the dev branch like you have master?

**Comment by sonofmun** [comment]
So that dipshits like me don't push directly to dev.

**Pull request merged**